data-access/persoon.ts

```diff
@@ -314,3 +322,44 @@ async function removeFractieFromCurrent(
 
   await update(deleteQuery);
 }
+
+async function setActiveMandatarissenEndDate(
+  id: string,
+  endDate: Date,
+): Promise<void> {
+  const escaped = {
+    persoonId: sparqlEscapeString(id),
+    endDate: sparqlEscapeDateTime(endDate),
+    dateNow: sparqlEscapeDateTime(new Date()),
+    beeindigdStatus: sparqlEscapeUri(MANDATARIS_STATUS.BEEINDIGD),
+  };
+  const updateQuery = `
+    PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+    PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+    DELETE {
+      ?mandataris mandaat:einde ?endDate.
+    }
+    INSERT {
+      ?mandataris mandaat:einde ${escaped.endDate}.
+    }
+    WHERE {
```

Comment:
When doing it in one sparql query it is easy to execute but as a user of the API you cannot provide usefull information of what happened. E.g. #updated, update that failed, updated mandataris ids that had an update. Now it's all magic and shows correct in the frontend 💭